### PR TITLE
Update contributing.rst

### PR DESCRIPTION
**Pull request by mimikaan**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by jreback** [comment]
this is duplicating #28705

File: doc/source/development/contributing.rst [hunk 1]
```diff
@@ -930,8 +930,12 @@ The ``pandas.util.testing`` module has many special ``assert`` functions that
 make it easier to make statements about whether Series or DataFrame objects are
 equivalent. The easiest way to verify that your code is correct is to
 explicitly construct the result you expect, then compare the actual result to
-the expected correct result::
+the expected correct result
```
Comment (jbrockmendel) [inline doc/source/development/contributing.rst]: period at the end
"writting" --> "writing"
"#Issue number ##NUMBER:" --> "# See GH#NUMBER"

**Comment by jbrockmendel** [comment]
@mimikaan can you address comments

File: doc/source/development/contributing.rst (hunk 1 above)
Comment (mimikaan) [reply doc/source/development/contributing.rst]: Alterations are correct

**Review by mimikaan — COMMENTED** [review]
alterations were correct

**Comment by jreback** [comment]
was already covered by https://github.com/pandas-dev/pandas/pull/28907

thanks

**Pull request closed without merging**